README.md

```diff
@@ -67,15 +76,23 @@ A private key has an accompanying public key:
 publicKey := privateKey.PublicKey()
 ```
 
-The example above uses an ECDSA key-pair of the elliptic curve P-256. Flow also supports the curve secp256k1. Here's how you can generate an `ECDSA-SECp256k1` private key:
+#### Supported Curves
+
+The example above uses an ECDSA key pair on the P-256 elliptic curve.
+Flow also supports the secp256k1 curve used by Bitcoin and Ethereum.
+
+Here's how to generate an ECDSA private key on the secp256k1 curve:
 
 ```go
 privateKey, err := crypto.GeneratePrivateKey(crypto.ECDSA_secp256k1, seed)
 ```
 
+Here's a full list of the supported signature and hash algorithms: [Flow Signature & Hash Algorithms](https://github.com/onflow/flow/blob/master/docs/accounts-and-keys.md#supported-signature--hash-algorithms)
```

Comment:
I was checking the linked page and I noticed SHA2-384 and SHA3-384 are included in the table. Was this a new update that was also implemented or is it just a mistake?

Reply:
Oops, that's a mistake in the linked documentation. I'll update there. Nice catch!